index.js

```diff
@@ -3,18 +3,11 @@
 const parse = require('./lib/xml')
 
 module.exports = (options) => {
-  return function * (next) {
-    const ctx = this
-    if (ctx.is('xml')) {
-      switch (ctx.method) {
-        case 'POST':
-        case 'PUT':
-        case 'PATCH':
-        case 'TRACE':
-        ctx.request.body = yield parse(ctx.req, options)
-        break
-      }
+    return function*(next) {
```

Comment:
when i use sublime text 3 to codeing,and then when save the file it auto format the code
i can't add space between function * (next)  ^O^